jsapi/externs/luxx.js

```diff
@@ -3,6 +3,59 @@
  */
 var luxx;
 
+/**
+ * Object literal with config options for the map.
+ * @typedef {{
+ *     bgLayer: (string),
+ *     layers: (Array<string>),
+ *     target: (Element|string),
+ *     position: (ol.Coordinate|undefined),
+ *     positionSrs: (string|undefined),
+ *     zoom: (number|undefined)
+ * }}
+ */
+luxx.MapOptions;
+
+/**
+ * Identifier of background layer. Default to `basemap_2015_global`.
+ * @type {string}
+ */
+luxx.MapOptions.prototype.bgLayer;
+
+/**
+ * Array of overlay layer identifiers.
+ * @type {Array<string>}
+ */
+luxx.MapOptions.prototype.layers;
+
+/**
+ * The container for the map, either the element itself or the `id` of the
+ * element.
+ * @type {Element|string}
+ * @api stable
+ */
+luxx.MapOptions.prototype.target;
+
+/**
+ * The initial center for the map view. The coordinate system for the center is
+ * specified with the `positionSrs` option.
+ * @type {ol.Coordinate|undefined}
+ */
+luxx.MapOptions.prototype.center;
+
+/**
+ * The projection of the position coordinates.
+ * Default is `2169` (Spherical Mercator).
+ * @type {string}
+ * @api stable
+ */
+luxx.MapOptions.prototype.positionSrs;
+
+/**
+ * Zoom level used to calculate the initial resolution for the view.
+ * @type {number|undefined}
+ */
+luxx.MapOptions.prototype.zoom;
```

Comment:
Any reason why `target` and `positionSrs` are api stable and no the other properties?